src/pages/calendar/components/AddTask/RadioButton.jsx

```diff
@@ -0,0 +1,74 @@
+import { useTranslation } from 'react-i18next'
+import styled, { css } from 'styled-components'
+
+export const RadioButton = ({ title, checked, onChange }) => {
+  const { t } = useTranslation();
+  let radioColor = '';
+  let priority=title.charAt(0).toUpperCase() + title.slice(1);
+
+  switch (title) {
+    case 'low':
+      radioColor = '114, 194, 248'
```

Comment:
1. you should use colors from theme
2. it's no good to find the colors here. If I will need your component in another plase for different items it will work incorrectly

Reply:
+